tests/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(GTEST_ROOT "${PROJECT_SOURCE_DIR}/third_party/gtest")
```

Comment:
I'm not sure if we need this, also I doubt it will work.

Reply:
We don't. I had done this for pull request upstream. It works in this case if you first build gtest in that directory (`cd third_party/gtest && cmake . && make`).

Reply:
ok